### PR TITLE
Update EMsg

### PR DESCRIPTION
**Pull request by xPaw**
Split into a separate PR as it removes a bunch of things.

This should be up to date with latest Steam client and javascript dumps.

### CODE REVIEW

**Comment by xPaw** [comment]
Build fails because ClientServiceMethodResponse and ClientServiceMethod have been renamed to Legacy.

**Comment by yaakov-h** [comment]
Removed items should be marked as `obsolete` for the first release to preserve compatibility for a single release, and only changed to `removed` afterwards to keep for historical purposes but bypass codegen.

**Comment by xPaw** [comment]
That has been done.

File: Resources/SteamLanguage/emsg.steamd [hunk 1]
```diff
@@ -13,9 +13,9 @@ enum EMsg
 	JobHeartbeat = 123;
 	HubConnect = 124;
 	Subscribe = 126;
-	RouteMessage = 127; removed
+	RouteMessage = 127;
 	RemoteSysID = 128; removed
-	AMCreateAccountResponse = 129;
+	AMCreateAccountResponse = 129; obsolete
```
Comment (yaakov-h) [inline Resources/SteamLanguage/emsg.steamd]: ?

File: Resources/SteamLanguage/emsg.steamd [hunk 2]
```diff
@@ -89,25 +96,29 @@ enum EMsg
...
-	AISUpdatePackageCostsResponse = 408;
-	AISCreateMarketingMessage = 409;
-	AISCreateMarketingMessageResponse = 410;
-	AISGetMarketingMessage = 411;
-	AISGetMarketingMessageResponse = 412;
-	AISUpdateMarketingMessage = 413;
-	AISUpdateMarketingMessageResponse = 414;
-	AISRequestMarketingMessageUpdate = 415;
-	AISDeleteMarketingMessage = 416;
+	AISAppInfoTableChanged = 407; obsolete
```
Comment (yaakov-h) [inline Resources/SteamLanguage/emsg.steamd]: ?

**Pull request merged**